BlueprintUI/Sources/Layout/Transform.swift

```diff
@@ -0,0 +1,134 @@
+import UIKit
+
+/// Changes the transform of the wrapped element.
+public struct Transform: Element {
+
+    /// The content element whose transform is being affected.
+    public var wrappedElement: Element
+
+    /// The transform of the wrapped element.
+    public var transform: CATransform3D
+
+    /// Initializes a `Transform` with the given content element and 3D transform.
+    ///
+    /// - parameters:
+    ///   - transform: The 3D transform to be applied to the wrapped element.
+    ///   - wrapping: The content element to be made transparent.
+    public init(
+        transform: CATransform3D,
+        wrapping wrappedElement: Element
+    ) {
+        self.transform = transform
+        self.wrappedElement = wrappedElement
+    }
+
+    /// Initializes a `Transform` with the given content element and DD transform.
+    ///
+    /// - parameters:
+    ///   - transform: The 2D transform to be applied to the wrapped element.
+    ///   - wrapping: The content element to be made transparent.
+    public init(
+        transform: CGAffineTransform,
+        wrapping wrappedElement: Element
+    ) {
+        self.transform = CATransform3DMakeAffineTransform(transform)
+        self.wrappedElement = wrappedElement
+    }
+
+
+    public var content: ElementContent {
+        return ElementContent(child: wrappedElement, layout: Layout(transform: transform))
+    }
+
+    public func backingViewDescription(bounds: CGRect, subtreeExtent: CGRect?) -> ViewDescription? {
+        return nil
+    }
+
+    private struct Layout: SingleChildLayout {
+        var transform: CATransform3D
+
+        func measure(in constraint: SizeConstraint, child: Measurable) -> CGSize {
+            return child.measure(in: constraint)
+        }
+
+        func layout(size: CGSize, child: Measurable) -> LayoutAttributes {
+            var attributes = LayoutAttributes(size: size)
+            attributes.transform = transform
```

Comment:
Do we want to do this; or would we want to wrap in another view that has a transform attached to it? Trying to think if this would end up "stomping over" any other transforms, like one from a transition...

Reply:
(Also, maybe Andrew can check me on this, but does this get applied correctly if the wrapped element in this case is also not view-backed, but instead just has several view backed child elements?)

Reply:
This looks right to me. Attributes set during layout are merged when the tree is flattened, and transition attributes are applied on top of the resolved attributes from layout, so I don't think there's any stomping.